frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx

```diff
@@ -21,7 +21,7 @@ export const Activity: React.FC<ActivityProps> = ({ timestamp, children }) => {
     <div className="co-activity-item__ongoing" data-test="activity">
       {timestamp && (
         <span className="text-secondary">
-          {t('dashboard~Started')}{' '}
+          {t('public~Started')}{' '}
```

Comment:
Use `console-shared` throughout this package.

```suggestion
          {t('console-shared~Started')}{' '}
```